<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(170, 255, 255);</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhNone</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="anh_tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">
border-radius: 5px; 
background-color: rgb(255, 255, 255);
border: 3px solid #000;

</string>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="vanban_tableWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>291</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(255, 255, 255);
border: 3px solid #000;</string>
    </property>
    <property name="alternatingRowColors">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="Quet_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 13pt;
font-weight: bold; 
border-radius: 5px; 
border: 2px solid #000; 
background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>Quét</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>anhmoi/scan.png</normaloff>anhmoi/scan.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="Huy_button">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 13pt;
font-weight: bold; ;
border-radius: 5px;  
border: 2px solid #000; 
background-color:rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>Hủy</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>anhmoi/cancel.png</normaloff>anhmoi/cancel.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="taianh_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 13pt;
font-weight: bold; 
border-radius: 15px;
border: 2px solid #000;
background-color: rgb(0, 85, 255);
</string>
    </property>
    <property name="text">
     <string>Tải Ảnh</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>anhmoi/upload.png</normaloff>anhmoi/upload.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lammoi_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 13pt;
font-weight: bold; 
border-radius: 5px;  
border: 2px solid #000;
background-color:rgb(255, 255, 0)</string>
    </property>
    <property name="text">
     <string>Làm Mới</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>anhmoi/refresh.png</normaloff>anhmoi/refresh.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="return_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">

font-size: 13pt;
font-weight: bold; 
border-radius: 5px;  
border: 2px solid #000;
background-color: rgb(255, 0, 0);
</string>
    </property>
    <property name="text">
     <string>Quay Lại</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>anhmoi/return.png</normaloff>anhmoi/return.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
